### DISCUSSION TITLE
"We noticed that you've recently submitted reports to AbuseIPDB on UDP traffic."

### DISCUSSION THREAD

**Discussion by oh6kw** [sefinek/UFW-AbuseIPDB-Reporter#2] [General]
Hello.

I got email from abuseipdb. Is there any option to prevent this UDP reports or shoud i allow all UDP in UFW?

_We noticed that you've recently submitted reports to AbuseIPDB on UDP traffic.

The offending reports were made against the IP address [193.210.xxx.xxx], and can be viewed [here](https://www.abuseipdb.com/account/reports?ip=193.210.xxx.xxx).

As a reminder, this is against our [reporting policy](https://www.abuseipdb.com/reporting-policy).

Sincerely,

The AbuseIPDB Team_

**Comment by sefinek** [comment]
Hi!

Yep, I received the same email yesterday.

Unfortunately, it's not possible at the moment.

I'm currently rewriting the script from Bash to Node.js because Bash isn't my main language. I know JavaScript isn't ideal for this kind of thing, but oh well... The functionality will remain the same, so no worries!

I'll also create a configuration file, `config.js`

**Comment by sefinek** [comment]
Hello again @oh6kw 

I have updated the script. UDP traffic will no longer be reported. This is the final update for the main branch. In the [node.js](https://github.com/sefinek/UFW-AbuseIPDB-Reporter/tree/node.js) branch, there's a new version that will soon replace the current one. The old version will still be available for installation.

~ https://github.com/sefinek/UFW-AbuseIPDB-Reporter/commit/9a997b2a5093b663fe2c3834771d1ce5e6593d47

JS version (beta):
<img width="821" alt="{2E7D0ABB-C698-4DEF-B658-FA461062FD50}" src="https://github.com/user-attachments/assets/d797b3c9-ce02-4ee6-902b-a7a9e5551295" />
<img width="899" alt="{86EF4BEB-F8DC-4206-9930-1D7DFB7A31F9}" src="https://github.com/user-attachments/assets/fccd11e9-e083-4523-9db4-fd7bddf0855e" />

**Comment by sefinek** [comment]
The version of the tool rewritten in Node.js has already been partially completed and is considered stable.
